plugins/org.eclipse.emf.mwe2.language.ui/.classpath

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins">
```

Comment:
i would rather prefer to have a PR in extras that adds mwe.ui as xfriends

Reply:
I agree. I removed the access restriction from the PR.